pkg/hintrunner/utils/math_utils.go

```diff
@@ -26,6 +26,24 @@ func EcDoubleSlope(pointX, pointY, alpha, prime *big.Int) (big.Int, error) {
 	return Divmod(n, m, prime)
 }
 
+func LineSlope(point_aX, point_aY, point_bX, point_bY, prime *big.Int) (big.Int, error) {
+	// https://github.com/lambdaclass/cairo-vm_in_go/blob/31c3628bc10ebc1628685b3cdfa72d0e938b533e/pkg/builtins/ec_op.go#L258
```

Comment:
I'd rather use the link of the function in the python VM, this is supposed to be implemented from there.
It would be // https://github.com/lambdaclass/cairo-vm_in_go/blob/31c3628bc10ebc1628685b3cdfa72d0e938b533e/pkg/builtins/ec_op.go#L258

Reply:
okay